index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<!-- HTML documents start with...
+	- a doctype
+	- <html> tag
+		- <head> tag
+			- css... maybe script tags
+		- <body> tag
+			- content...
+  -->
+<html>
+	<head>
+	</head>
+	<body>
+		<p>This is a paragraph tag.</p>
+		<div>This is a div</div>
+
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.js"></script>
+		<script>
+			console.log("I just loaded jQuery!");
```

Comment:
Hey, you had an empty `index.html`. I'm just helping you out.

I might have some suggestions for `for loops` and stuff, too.